src/feedback/trulens/feedback/v2/feedback.py

```diff
@@ -155,13 +204,13 @@ class Groundedness(Semantics, WithPrompt):
     # hugs._summarized_groundedness
     # hugs._doc_groundedness
 
-    system_prompt: ClassVar[PromptTemplate] = PromptTemplate.from_template(
+    system_prompt: ClassVar[str] = cleandoc(
```

Comment:
Removing langchain as a dependency for `trulens-feedback` so prompts are back to classic strings.